rust/main/agents/relayer/src/settings/mod.rs

```diff
@@ -307,6 +310,12 @@ impl FromRawConf<RawRelayerSettings> for RelayerSettings {
             })
             .unwrap_or_default();
 
+        let max_message_retries = p
+            .chain(&mut err)
+            .get_opt_key("maxMessageRetries")
```

Comment:
this will have to be set as `HYP_MAX_MESSAGE_RETRIES`

Reply:
not super important but would be `HYP_MAXMESSAGERETRIES` right?